.changeset/spicy-garlics-switch.md

```diff
@@ -0,0 +1,12 @@
+---
+"rxfire": patch
```

Comment:
IMO this is a patch since it's cleaning up poorly defined behavior and bugs and this library is in beta.